storm-core/src/ui/public/flux.html

```diff
@@ -0,0 +1,139 @@
+<html>
```

Comment:
Added a link on the top right hand side after the username place holder. Here's the icon:
[Icon](https://github.com/ambud/storm/blob/af8ed339cb3896100dabe3619c0fba2d19165dff/storm-core/src/ui/public/images/flux.png)

Y for YAML and also for Flux Capacitor from Back to the future 💯 

Please let me know if this works.